ui/src/preview/preview-view.tsx

```diff
@@ -14,22 +14,25 @@ import {
 import { PreviewFooter } from 'preview/preview-footer';
```

Comment:
Pitäisi varmaan pilkkoa tätä tiedostoa useampaan osaan. Melkoisesti varsinkin tyyppimäärityksiä päätyi nyt tänne.

Reply:
Tyypitykset voinee pistää `publication-model.ts`:ään (tai sitten voi tehdä oman tiedostonkin noille, onhan noita aika paljon täällä näköjään. Up yours että miten haluat noiden kanssa toimia). Ne kuitenkin liittyvät

Reply:
Mjeps, sinänsä moni näistä nykyisistä/uusista tyypityksistä on vaan apukäytössä tässä esikatselunäkymässä, eikä niinkään yleistä julkaisumallia. Pitänee miettiä jokin järkevä paikka taikka uusikin tiedosto tosiaan 🤔

Reply:
👍 Jaoin tuon vähän pienempiin osiin jo tässä vaiheessa. Nyt löytyy preview-data.ts, review-view-filters.ts ja preview-view-revert-request.ts. Nuo -data.tsx ja -filters.ts ovat nimityksenä vähän niin ja näin, mutta ne elävät todennäköisestim muutenkin vielä sen toisen tiksun refakin yhteydessä.